shared/general/config-variables-upboard.md

```diff
@@ -0,0 +1,3 @@
+Name | Default | Description | Format
+--- | --- | ---
+BALENA_HOST_CONFIGFS_ssdt | "" | Which SSDT overlays should be loaded | `value | "value" | "value1","value2"`
```

Comment:
@richbayliss we need to mention what version of the supervisor will support this. I think the other variables all have supervisor version there too.